pkg/internal/codegen/parse/util.go

```diff
@@ -231,6 +231,17 @@ func hasSingular(t *types.Type) bool {
 	return false
 }
 
+// hasRequired returns true if t is annotated with
+// +required
+func hasRequired(t *types.Type) bool {
+	for _, c := range t.CommentLines {
```

Comment:
do we still have SecondCommentLines? We would need that as well here.

Reply:
`SecondClosestCommentLines`? We can check that too, do we need to check both of them?

Reply:
I think yes. The `SecondClosestCommentLines` was invented to exclude those tags from generated documentation.